aw_server/server.py

```diff
@@ -60,8 +60,6 @@ def create_app(
 @root.route("/")
 def static_root():
     return current_app.send_static_file("index.html")
-    return send_from_directory("/", "index.html")
-
```

Comment:
There should be two newlines between functions, otherwise all looks good!

Reply:
Added an extra line!